### PR TITLE
Hide filtered alerts

### LINKED ISSUE #2: Hide filtered alerts
Currently the filtered alerts are shown with information which filter had been applied.  This helps the development of the code and development of the configuration, but overloads the interface.

The filtered list could be hidden behind a configuration item or personal configuration or some other method.

### PR DESCRIPTION
**Pull request by greenhalos**
1. Button to show or hide the filtered alerts
2. State is stored in the localstorage of the browser (default is hidden)
3. Counter to show how many alerts are hidden

image

Resolves #2

### CODE REVIEW

File: pkg/web/templates/alerts.gohtml [hunk 1]
```diff
@@ -37,9 +37,11 @@
...
 </table>
 
-<br /><br /><br /><br /><br /><br /><br />
-<h3 class="filtered">Filtered Alerts</h3>
-<table class="widetable filtered">
+<h3 class="filtered">
+    Filtered Alerts ({{ len .Content.Blocked }})
+    <button id="toggle-filtered-alerts">Show</button>
+</h3>
+<table id="filtered-table" class="widetable filtered hidden">
```
Comment (BuJo) [inline pkg/web/templates/alerts.gohtml:44]: Unsure how it interacts with the reload mechanic.  Would it stay open if the backend sends new data?  (new data means that the full `alerts.gohtml` is sent to the frontend and replaces the previous content.

Comment (BuJo) [reply pkg/web/templates/alerts.gohtml:44]: Idea: Maybe `turbo:before-render` could be used to toggle it open again on reload.
https://turbo.hotwired.dev/reference/events

I guess.. we could use the same mechanic for-reopening items as well.
There is no way to distinguish rows right now, but for silencing there is an ID available.  We could use `aria-data` or something to make rows accessible via the ID.

Comment (greenhalos) [reply pkg/web/templates/alerts.gohtml:44]: I will have a look into it :)

Comment (BuJo) [reply pkg/web/templates/alerts.gohtml:46]: For now I've made it possible that the state is preserved when the site is reloaded.  It works by simply re-registering the handler after the stream rendered.

**Review by BuJo — APPROVED** [review]
I've tested it.  It seems to work.

**Pull request merged**